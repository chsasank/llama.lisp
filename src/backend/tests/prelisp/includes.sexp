(c-lisp
    ,@(add_print_includes)

    (define ((main int))
        (if #t
            ((call print 1)
             (call print 2)))

        (if #f
            ((call print 0))
            ((call print 3)))
             (ret 0)))
