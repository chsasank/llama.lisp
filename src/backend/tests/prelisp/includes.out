(c-lisp
  (define ((print int) (n int)))
  (define ((fprint float) (n float)))
  (define ((main int))
    (if #t ((call print 1) (call print 2)))
    (if #f ((call print 0)) ((call print 3)))
    (ret 0)))
